### ISSUE TITLE
Don't make us wait forever when starting a docker/jupyterlab fails.

### ISSUE THREAD

**Issue by rokroskar** [SwissDataScienceCenter/renku-notebooks#62]
_From @erbou on September 28, 2018 10:22_

**Is your feature request related to a problem? Please describe.**
The starting Jupyterlab will not notify me if something went wrong, and can make me wait forever.

**Describe the solution you'd like**
Give the option to get more details (can be summarized, no need for a full log) about what it is doing, and what's left to do, with a brief status (Ok, Fail) of operation, such as:
```
* [Ok]   starting docker container
* [Fail]  cloning repo
* [    ]   importing data   <- for git submodule update / git lfs pull
```

**Describe alternatives you've considered**
At a minimum we should report an error, so that know when there's no point to keep waiting. 

_Copied from original issue: SwissDataScienceCenter/renku-ui#325_

**Comment by rokroskar** [comment]
At the moment, this is actually done in the notebooks service. I will move the issue there.

**Comment by lorenzo-cavazzi** [comment]
This is *partially* addressed by #253 since the launching pods get evicted after a timeout instead of pending there forever.

Implementing a better mechanism (as suggested here SwissDataScienceCenter/renku-ui#787) is not that trivial. A possible solution is this one jupyterhub/jupyterhub#2975
We would probably need to implement that solution by ourselves, it shouldn't take too long, but it would be nice to get a feedback first since it may not be the optimal solution.

**Comment by olevski** [comment]
I think that in light of https://github.com/SwissDataScienceCenter/renku-ui/pull/1605 this should be closed.
